packages/ui-kit/package.json

```diff
@@ -32,14 +32,19 @@
     "lodash.escape": "4.0.1",
     "lodash.throttle": "4.1.1",
     "prism-react-renderer": "1.2.1",
-    "prop-types": "15.8.1",
     "react-intl": "^5.7.0",
+    "react-is": "^17.0.2",
     "swr": "1.1.2",
     "tiny-invariant": "1.2.0"
   },
   "devDependencies": {
+    "@svgr/cli": "6.2.0",
+    "@types/react": "^17.0.38",
+    "@types/react-dom": "^17.0.11",
+    "@types/react-is": "^17.0.3",
     "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "react-dom": "17.0.2",
+    "typescript": "^4.5.5"
```

Comment:
There should be no need for this, as it's already defined in the root package.json.